controls/pcidss_4.yml

```diff
@@ -574,6 +574,8 @@ controls:
       levels:
         - base
       status: automated
+      notes: |-
+        Related to requirement 12.3.3.
       rules:
         - configure_crypto_policy
```

Comment:
Do we want to add `package_telnet-server_removed` here or something similar?

Reply:
Yes, it is already selected in `2.2.4`.